src/go/intelligence/int_test/test_wrapper.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+##H Script for automation of testing process of the intelligence module.
+
+#Variables
+CMSMONITORING_REPO=$(pwd)/CMSMonitoring
```

Comment:
Please do not use pwd. Instead write script to accept input parameter about location of the script. For example
```
# check that user provides input parameter
if [ $# -ne 1 ]; then
   echo "Usage: ..."
fi
# use input parameter to setup workdir
wdir=$1
....
then in you script use $wdir.
```